src/components/auth/profile/DonationTab.tsx

```diff
@@ -28,11 +28,18 @@ import { ProfileTabs } from './tabs'
 import ProfileTab from './ProfileTab'
 import DonationTable from './DonationTable'
 
-const useStyles = makeStyles({
-  donationsBox: {
+const PREFIX = 'DonationTab'
+
+const classes = {
+  donationsBox: `${PREFIX}-donationsBox`,
+  donationsBoxRow: `${PREFIX}-donationsBoxRow`,
+}
+
+const StyledProfileTab = styled(ProfileTab)({
+  [`& .${classes.donationsBox}`]: {
```

Comment:
Is this the way we should do class styling now?

Reply:
This is the automated JSS to styled migration

We can minimize the usage of styled as much as possible with `sx` property and reusable components instead of reusable styles.

I'm not a big fan of the way styles are re-defined, so I did take an initiative the change them after the migration to local `sx` props, but it's too much to be handled at once